lib/src/client/mod.rs

```diff
@@ -64,6 +69,13 @@ pub struct PlayCallTransactionData {
     target_account_id: AccountId,
 }
 
+#[derive(Clone)]
+pub struct PlayFoldTransactionData {

```

Comment:
Shall we start moving types to `types` library crate ?

Reply:
Yes ig